Sources/StreamVideo/OpenApi/generated/Models/UserMutedEvent.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+    
+public final class UserMutedEvent: @unchecked Sendable, Event, Codable, JSONEncodable, Hashable {
```

Comment:
Is this again for chat? We received audio/video muted events in a different way.